test/enhance/test_adjust.py

```diff
@@ -742,3 +742,57 @@ def build_input(channels, depth, height, width, bs=1, row=None):
         batch = torch.stack([image3d] * bs)
 
         return batch
+
+
+class TestSharpness(object):
+
+    f = kornia.enhance.sharpness
+
+    @pytest.mark.parametrize("channels", [1, 3, 5])
+    def test_shape(self, channels, device, dtype):
+        bs, height, width = 1, 4, 5
+
+        inputs = torch.ones(channels, height, width, device=device, dtype=dtype)
+
+        assert TestSharpness.f(inputs, 0.8).shape == torch.Size([bs, channels, height, width])
+
+    @pytest.mark.parametrize("channels", [1, 3, 5])
+    def test_shape_batch(self, channels, device, dtype):
+        bs, height, width = 2, 4, 5
+
+        inputs = torch.ones(bs, channels, height, width, device=device, dtype=dtype)
+
+        assert TestSharpness.f(inputs, 0.8).shape == torch.Size([bs, channels, height, width])
+
+    def test_equalize(self, device, dtype):
+        torch.manual_seed(0)
+
+        inputs = torch.rand(1, 3, 3).to(device=device, dtype=dtype)
+
+        expected = torch.tensor([
```

Comment:
Could you add a comment about how did you get those numbers. Are they from opencv?